packages/common/src/eventSender.ts

```diff
@@ -0,0 +1,3 @@
+import type * as _EventSender from '@tidal-music/event-producer';
```

Comment:
Common should not be allowed to import from any SDK module I think, or else we create circular dependencies?

Reply:
For type improts it's fine!